app/src/organisms/CommandText/PipettingCommandText.tsx

```diff
@@ -18,16 +14,17 @@ import type {
   PipetteName,
   PipettingRunTimeCommand,
 } from '@opentrons/shared-data'
+import type { CommandTextData } from './types'
 
 interface PipettingCommandTextProps {
   command: PipettingRunTimeCommand
-  robotSideAnalysis: CompletedProtocolAnalysis
+  protocolData: CommandTextData
```

Comment:
nit
I guess we use `protocolData` in some components so maybe using a different name would be easy to understand?

```suggestion
  commandTextData: CommandTextData
```